eventually/tests/unittests/validators/testvalidators.py

```diff
@@ -124,6 +133,627 @@ def test_password_validator_fail(self):
 
         self.assertIsNone(is_valid)
 
+    def test_password_validator_type_error_fail(self):
+        """
+        Method that tests `password_validator`,
+        when `password` parameter has wrong type (not string).
+        """
+        password = 12
+
+        is_valid = password_validator(password)
+
+        self.assertIsNone(is_valid)
+
+    def test_login_validate_success(self):
+        """
+        Method that tests `login_validate`.
+        """
+        data = {
+            "email": "example@gmail.com",
+            "password": "some_password"
```

Comment:
should this password be valid?

Reply:
It does not matter for test_login_validate_success, because it does not check 'password' value here.